fluss-flink/fluss-flink-tiering/pom.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2025 Alibaba Group Holding Ltd.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.alibaba.fluss</groupId>
+        <artifactId>fluss-flink</artifactId>
+        <version>0.7-SNAPSHOT</version>
+    </parent>
+
+    <packaging>jar</packaging>
+
+    <artifactId>fluss-flink-tiering</artifactId>
+    <name>Fluss : Flink : Tiering</name>
+
+    <properties>
+        <flink.minor.version>1.20.1</flink.minor.version>
```

Comment:
big question: do we need to support multiple flink versions as we support multiple-versions' flink connector?

Reply:
Yes, we should and it can support. The `flink.minor.version` in here is just for testing purpose. Since our core flink tiering code is in fluss-flink-common. So, if user uses flink-1.18, they should use fluss-flink-1.18 jar as well as this `fluss-flink-tiering` jar.  `fluss-flink-tiering` jar just provide a main method to enable Flink to run this jar.